source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -198,6 +198,9 @@ bool ShaderRenderTester::validate(const mx::FilePathVec& testRootPaths, const mx
     // Set target unit space
     context.getOptions().targetDistanceUnit = "meter";
 
+    // Register shader metadata defined in the libraries.
+    _shaderGenerator->registerShaderMetadata(dependLib, context);
```

Comment:
@niklasharrysson , this is the only interesting difference that was in ILM master but not here.